### DISCUSSION TITLE
Aile dizilimi sorusu

### DISCUSSION THREAD

**Discussion by sonsuzus** [sonsuzus/sonsuzus#280] [Zeka Sorusu] [answered]
Dedeniz, babanız, amcanız, siz, kardeşiniz ve amcanızın iki çocuğu olmak üzere toplam 7 kişisiniz. Hiç kimsenin babasından önce gelmemesi şartıyla kaç farklı şekilde sıralanabilirsiniz?

**Comment by AcelyaP** [comment, marked as answer]
**Kara olanlar sabit**

**D A B** 4 3 2 1 çarptığımız 24 çıkıyor o yüzden 24 kez
**D A** 2(kuzenden birisi) **B** 3 2 1 hepsini çarptığında 12 o yüzden 12 kez
**D A** 2 1 **B** 2 1 çarptığımızda 4 oluyor o yüzden 4 kez
**D B A** 4 3 2 1 çarptığımızda da 24 çıkıyor o yüzden 24 kez
**D B** 2(ben yada kardeşim) **A** 3 2 1 hepsini çarptığında 12 o yüzden 12 kez
**D B** 2 1 **A** 2 1 çarptığımızda 4 oluyor o yüzden 4 kez

toplam 80 o yüzden cevap 80'dir.
